doc/ranger.pod

```diff
@@ -302,6 +302,20 @@ The macro %space expands to a space character. You can use it to add spaces to
 the end of a command when needed, while preventing editors to strip spaces off
 the end of the line automatically.
 
+The macros %any and %any_path can be used when binding keys to
```

Comment:
Could you try summarizing better what it might be used for? Maybe "When mapping keys you can use the placeholder `<any>`, the key entered in that position can be used through the `%any` and `%any_path` macros...